pkgs/applications/audio/qpwgraph/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, mkDerivation, fetchFromGitLab
+, cmake, pkg-config
+, alsa-lib, pipewire
+, ...
+}:
+
+mkDerivation rec {
+  pname = "qpwgraph";
+  version = "0.0.6";
+
+  src = fetchFromGitLab {
```

Comment:
Great to see that you are using the gitlab fetcher.

Reply:
With recent github problems this sounds very ironic :^)